client/transaction-pool/graph/src/validated_pool.rs

```diff
@@ -209,8 +208,11 @@ impl<B: ChainApi> ValidatedPool<B> {
 				self.rotator.ban(&Instant::now(), removed.iter().map(|x| x.clone()));
 				removed
 			};
+			if removed.len() > 0 {
```

Comment:
```suggestion
			if !removed.is_empty() {
```

Reply:
Maybe merge this with previous log statement (which is also `debug`)? And just print extra information about removed entries always.